.circleci/config.yml

```diff
@@ -33,20 +33,15 @@ jobs:
             echo "CIRCLE_REPOSITORY_URL: ${CIRCLE_REPOSITORY_URL}"
             echo $CIRCLE_JOB > circlejob.txt
 
-      - run:
-          name: 🔎 Unit tests
-          command: |
-            julia test/ci_prepare.jl
-
       - run:
           name: ⚙️  Integration tests
           command: |
+            julia --project -e 'import Pkg; Pkg.instantiate(); Pkg.update();'
```

Comment:
N.B. we now install the Julia deps in the repo environment as we now assume that the Julia test runner uses `julia --project` internally following https://github.com/plotly/dash/pull/2573